var/spack/repos/builtin/packages/perl-parsetemplate/package.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PerlParsetemplate(PerlPackage):
```

Comment:
The naming convention for metacpan-based perl packages implies that the directory for this package should be `perl-parse-template` and the class name `PerlParseTemplate`.

Reply:
I tend to disagree here.
If you look at e.g. [PathTools](https://metacpan.org/dist/PathTools), we base the Spack name on the distribution name. This makes sense, otherwise we'd have to pick if this should be perl-cwd or perl-file-spec.
Given that, the distname is ParseTemplate, the Spack name should be perl-parsetemplate.